Ical.Net/Utility/DateUtil.cs

```diff
@@ -63,8 +62,11 @@ private static Dictionary<string, string> InitializeWindowsMappings()
     /// <param name="tzId">A BCL, IANA, or serialization time zone identifier</param>
     /// <param name="useLocalIfNotFound">If true, this method will return the system local time zone if tzId doesn't match a known time zone identifier.
     /// Otherwise, it will throw an exception.</param>
+    /// <exception cref="ArgumentException">Unrecognized time zone id</exception>
     public static DateTimeZone GetZone(string tzId, bool useLocalIfNotFound = true)
     {
+        var exMsg = $"Unrecognized time zone id {tzId}";
+
         if (string.IsNullOrWhiteSpace(tzId))
```

Comment:
We removed the implicit usage of the local time zone in other parts, didn't we? Hope, this is the only one left.